### PR TITLE
NPM with full data

### PR DESCRIPTION
**Pull request by robertbastian**

### CODE REVIEW

File: ffi/diplomat/js/examples/wasm-demo/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "clean": "rm dist/*",
     "build": "webpack",
-    "start": "webpack serve --mode development",
+    "start": "webpack serve --mode development --port 12345",
```
Comment (sffc) [inline ffi/diplomat/js/examples/wasm-demo/package.json:9]: ?

File: ffi/diplomat/js/examples/wasm-demo/src/ts/date-time.ts [hunk 2]
```diff
@@ -111,7 +111,7 @@ export class DateTimeDemo {
                 this.#displayFn("");
             }
         } catch (e) {
-            this.#displayFn(`Error: ${e.error_value}`);
+            this.#displayFn(`Error: ${e}`);
```
Comment (sffc) [inline ffi/diplomat/js/examples/wasm-demo/src/ts/date-time.ts:114]: We should probably change this back

File: tools/scripts/ffi.toml [hunk 3]
```diff
@@ -165,8 +165,6 @@ script = '''
 cd ffi/diplomat/js/examples/node
 exec --fail-on-error npm ci
 exec --fail-on-error npm test
-cd ../fixed_decimal
```
Comment (sffc) [inline tools/scripts/ffi.toml:168]: We should probably make this work too

**Review by sffc — APPROVED** [review]

**Review by Manishearth — APPROVED** [review]

File: ffi/diplomat/js/examples/wasm-demo/src/ts/date-time.ts (hunk 2 above)
Comment (robertbastian) [reply ffi/diplomat/js/examples/wasm-demo/src/ts/date-time.ts:114]: I don't think we should, this makes it easier to debug, same reason why we turned off minification.

File: tools/scripts/ffi.toml (hunk 3 above)
Comment (sffc) [reply tools/scripts/ffi.toml:168]: ok, I think it's fine for `wasm-demo` itself to serve as the test that the npm package is working

**Pull request merged**